decidim-proposals/spec/models/decidim/proposals/abilities/current_user_ability_spec.rb

```diff
@@ -41,7 +41,7 @@
         }
       end
 
-      it { is_expected.not_to be_able_to(:unvote, proposal) }
+      it { is_expected.not_to be_able_to(:vote, proposal) }
```

Comment:
Why is this changed? Same for next spec.

Reply:
I think it was a mistake when writing the spec because it states "when voting is disabled it is expected not to be able to unvote/?/vote the proposal" I would bet for vote.
Same for next spec.